src/sif/CMakeLists.txt

```diff
@@ -7,7 +7,6 @@ set(sources
   motorcyclecost.cc
   motorscootercost.cc
   nocost.cc
-  osrm_car_duration.h
```

Comment:
in the diff i dont see the file being actually  moved into the headers directory, how is this building?

Reply:
nevermind for some reason i just didnt notice it!